src/mtx/CommandBinaryBuffer.ts

```diff
@@ -1194,6 +1232,31 @@ class CommandBinaryBuffer extends BinaryBuffer {
         this.setUint32(saldoParameters.powerThreshold);
         this.setInt32(saldoParameters.creditThreshold);
     }
+
+    getPeriods ( periodsNumber:number ): Array<IPeriod> {
+        const firstPeriod = this.getUint16();
+
+        if ( firstPeriod === 0xffff ) {
```

Comment:
we shouldn't stop if firstPeriod is 0xfffff

Reply:
fixed